assets/js/translator.js

```diff
@@ -1242,6 +1265,43 @@ function autoSelectDstLang() {
     $('#dstLangSelect').val(curDstLang).change();
 }
 
+// Check if the Language pair Exist or Not
+function checkLangPairAvailable(lang) {
+    if(languages[lang] !== undefined) {
+        curSrcLang = iso639CodesInverse[lang];
+        autoSelectDstLang();
```

Comment:
A function called `checkLangPairAvailable` should not have any side effects, i.e. it shouldn't be flapping around the state of the UI by changing state.

Reply:
Will it create side effects?

Reply:
Okay working on it.:)

Reply:
Done. I think! :)